sys/include/net/gnrc/lwmac/hdr.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *               2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_lwmac
+ * @{
+ *
+ * @file
+ * @brief       Header definition LWMAC
+ * @internal
+ * @author      Daniel Krebs <github@daniel-krebs.net>
+ * @author      Shuguo Zhuo  <shuguo.zhuo@inria.fr>
+ */
+
+#ifndef GNRC_LWMAC_HDR_H_
+#define GNRC_LWMAC_HDR_H_
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "net/ieee802154.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   LWMAC WR frame type
+ */
+#define FRAMETYPE_WR             (0x01U)
```

Comment:
For all macros, types and functions in these headers. Please prefix with `GNRC_LWMAC` or `gnrc_lwmac` respectively.